src/DeprecatedCallableInfo.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class DeprecatedCallableInfo
```

Comment:
I'm not really sure about that, but is it really a callable? 
What about `DeprecatedInfo` only, that would match `deprecated_info` option?

Reply:
It is the info about a deprecated callable.

Reply:
Recently, I've created an abstraction on top of filters/functions/tests that I called a Twig callable :)
Filters/functions/tests are an abstraction on top of a PHP callable.

Reply:
See the AbstractTwigCallable class

Reply:
Oh okay, that makes sense then, thanks :)